queu.h

```diff
@@ -0,0 +1,37 @@
+#ifndef QUEU_H_INCLUDED
+#define QUEU_H_INCLUDED
+#include"obj/conf.h"
+#ifdef linked
+typedef struct node {
+    int key;
+    struct node* next;
+}node;
+
+typedef struct queue {
+    int size;
+    struct node *front, *rear;
+}queue_t;
+ static node*newnode(int val);
+ queue_t* queue_initialize(queue_t*q);
+ void queue_enqueue(queue_t* q, int val);
+ void queue_dequeue(queue_t* q);

```

Comment:
if this function won't return anything then you have to make another function to get the value